api/dp-openapi.yaml

```diff
@@ -393,6 +393,13 @@ paths:
         required: true
         schema:
           type: string
+      - name: field
+        description: Field to return, if not set all fields are returned.
+          To request multiple fields, append with [] and pass one per desired
+          field (e.g. field[]=projectid&field[]=author).
+        in: query
+        schema:
+          type: string
```

Comment:
ideally we could identify the fields to be requested here as some enumeration

Reply:
That's a good point. It makes the yaml even more verbose and redundant, but I guess that can't be helped.

Reply:
It's a good idea but I'd punt on this for now. We don't enumerate them for the other endpoints and someone can request the data and then pare down via fields.